lib/i18n/app_de.json

```diff
@@ -0,0 +1,20 @@
+{
+  "common": {
+    "appName": "Grüne App"
+  },
+  "news": {
+    "news": "Artikel"
+  },
+  "campaign": {
+    "campaign": "Wahlkampf"
+  },
+  "profile": {
+    "profiles": "Mitglieder"
+  },
+  "twoFactorAuthentication": {
+    "twoFactorAuthentication": "2FA"
+  },
+  "tools": {
+    "tools": "Tools"
+  }
```

Comment:
These should probably align with the folder/feature names in #249. I'll update those once we decided on the names.